chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -3513,22 +3513,9 @@ impl Get<u32> for ConfigurationCollatorRotationSessionPeriod {
     }
 }
 
-// CollatorAssignment expects to set up the rotation's randomness seed on the
-// on_finalize hook of the block prior to the actual session change.
-// So should_end_session should be true on the last block of the current session
-pub struct BabeGetRandomnessForNextBlock;
-impl GetRandomnessForNextBlock<u32> for BabeGetRandomnessForNextBlock {
-    fn should_end_session(n: u32) -> bool {
-        // Check if next slot there is a session change
-        n != 1 && {
-            let diff = Babe::current_slot()
-                .saturating_add(1u64)
-                .saturating_sub(Babe::current_epoch_start());
-            *diff >= Babe::current_epoch().duration
-        }
-    }
-
-    fn get_randomness() -> [u8; 32] {
+pub struct BabeGetCollatorAssignmentRandomness;
```

Comment:
let's try to put common structs in common places. in this case both starlight and dancelight use them, so likely we can use somewhere a common crate for these